### PR TITLE
Move internal/x/grpc to internal/grpc

### PR DESCRIPTION
**Pull request by peter-edge**

### CODE REVIEW

File: internal/grpc/handler.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	}
+	fileDescriptorSet, err := desc.SortFileDescriptorSet(service.FileDescriptorSet, service.FileDescriptorProto)
+	if err != nil {
+		return nil, err
+	}
+	return grpcurl.DescriptorSourceFromFileDescriptorSet(fileDescriptorSet)
+}
+
+func getServiceForMethod(method string) (string, error) {
+	split := strings.Split(method, "/")
```
Comment (AlexAPB) [inline internal/grpc/handler.go]: Is `package.service/Method` a gRPC standard or just a `grpcurl` standard? If it's just `grpcurl` how would you feel if we separated these into two fields instead? `... package.service Method ...`

File: internal/grpc/invocation_event_handler.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	"github.com/golang/protobuf/proto"
+	"github.com/jhump/protoreflect/desc"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+var jsonpbMarshaler = &jsonpb.Marshaler{Indent: "  "}
+
+type invocationEventHandler struct {
```
Comment (AlexAPB) [inline internal/grpc/invocation_event_handler.go]: Can we add a static check with the `grpcurl.InvocationEventHandler` or a comment to indicate that we're implementing the below methods to match that interface.

File: internal/grpc/invocation_event_handler.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+func (i *invocationEventHandler) OnReceiveResponse(message proto.Message) {
+	i.println(i.marshal(message))
+}
+
+func (i *invocationEventHandler) OnReceiveTrailers(s *status.Status, _ metadata.MD) {
+	if err := s.Err(); err != nil {
+		// TODO(pedge): not great for streaming
+		i.err = err
+		// printed by returning the error in handler
+		//i.println(err.Error())
```
Comment (AlexAPB) [inline internal/grpc/invocation_event_handler.go]: drop the comment

File: internal/grpc/invocation_event_handler.go [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+func (i *invocationEventHandler) OnReceiveHeaders(metadata.MD) {}
+
+func (i *invocationEventHandler) OnReceiveResponse(message proto.Message) {
+	i.println(i.marshal(message))
+}
+
+func (i *invocationEventHandler) OnReceiveTrailers(s *status.Status, _ metadata.MD) {
+	if err := s.Err(); err != nil {
+		// TODO(pedge): not great for streaming
```
Comment (AlexAPB) [inline internal/grpc/invocation_event_handler.go]: why not :)

File: internal/grpc/handler.go (hunk 1 above)
Comment (peter-edge) [reply internal/grpc/handler.go]: It’s a gRPC standard, the request url is /package.service/method

File: internal/grpc/invocation_event_handler.go (hunk 2 above)
Comment (peter-edge) [reply internal/grpc/invocation_event_handler.go]: Done

File: internal/grpc/invocation_event_handler.go (hunk 3 above)
Comment (peter-edge) [reply internal/grpc/invocation_event_handler.go]: Done

File: internal/grpc/invocation_event_handler.go (hunk 4 above)
Comment (peter-edge) [reply internal/grpc/invocation_event_handler.go]: Yea I guess it's fine

**Pull request merged**